docs/guides/haiku_migration_guide.rst

```diff
@@ -713,4 +713,152 @@ Finally, let's quickly view how the ``RNN`` Module would be used in both Haiku a
 
 The only notable change with respect to the examples in the previous sections is that
 this time around we used ``hk.without_apply_rng`` in Haiku so we didn't have to
-pass the ``rng`` argument as ``None`` to the ``apply`` method.
\ No newline at end of file
+pass the ``rng`` argument as ``None`` to the ``apply`` method.
+
+Scan Over Layers
+----------------
+One very important application of ``scan`` is apply a sequence of layers iteratively
```

Comment:
There's `jax.lax.scan` and `jax.linen.scan`, so let's use full API names. Also a small grammar fix.

```suggestion
One very important application of ``flax.linen.scan`` is to apply a sequence of layers iteratively
```